packages/design-system/adr/process-ds-transformation.md

```diff
@@ -0,0 +1,87 @@
+# Transform Design system process
+
+## Current status of DS?
+
+- A dedicated team → 2 DS dev + 1 DS designer
+- In _collaboration_ with products designers/content designers
+- _Lack of collaboration_ from product developers
+- _Low usage_ in products
+- How can we move from the DS team to something?
+
+## Org proposals
+
+- Have a composite team from the different domains (1 or 2 devs per domain) and Designers => `Design System Champions 🎖`
+
+  - Goal : Bring the product teams inside the process of build the DS
+
+  - Existing teams (nb of frontend dev):
+
+    - API : Tester / Designer / Creator (1,5?)
+    - DGD : Data Preparation / Data Stewardship / Data Inventory (4,5)
+    - DPD : Pipeline Designer (3)
+    - TDOPS : TMC / Portal / IDAAS (7)
+    - STITCH ?
+
+  - Timeslot Placeholder DS in domains capacity _(see TBD section)_
+    - Jira:
+      - "Main DS ticket" would stay on `TUx`
+      - Tickets would be created in the JIRA's products that take them and be linked to the TUx
+
+## How to construct a roadmap
+
+Discuss the roadmap:
+
+- Activity with priority 1
+
+  - Create new components
+  - Update existing components in DS (not those in TUI)
+
+- Activity with priority P2 (background tasks, less priority)
+  - Integrate & migrate existing components (TUI) to the DS
+  - Organise products migration
+
+## Process
+
+- 1 weekly meeting:
+
+  - To discuss of subjects: 1 squad / subject with a small team composed with dev and UX designer (and Content designer when it's needed)
+  - The purpose of this meeting would be to get a quick round of synchronisation and to discuss about the next steps, but not going into details.
+
+- 1 monthly meeting
+
+  - To share progress, organise internal roadmap (manage subject and squad) and construct roadmap who will be shared with product teams.
+  - How to prepare this meeting?
+
+    - There would be a need to have someone to animate the agenda
+    - Manage a board:
+      > Jira? github project?
+    - Do we need UX designer?
+      > YES
```

Comment:
```suggestion
    - Do we need UX designer?
      > YES
```
You need them all. 

A design system is a dictionary of UI patterns (even if we pretend it's UX). It's their job to design UIs, ours to technically challenge then implement them. Having been through a previous experience where the DS was engineer-led, I guarantee you it'll quickly devolve into a component libraries festival :D

Involve them, as soon as possible, in the discussions about the next steps without a dedicated team. Trust them.